### DISCUSSION TITLE
consider exposing worker_connections as a config setting

### DISCUSSION THREAD

**Discussion by lukesmurray** [tiangolo/meinheld-gunicorn-flask-docker#156] [Questions] [answered]
[Meinheld reads](https://github.com/mopemope/meinheld/blob/6ddb22b52783d22ba15c6cdfbcbc9ed05b5c0c46/meinheld/gmeinheld.py#L40)   | [gunicorn config `worker_connections`](https://docs.gunicorn.org/en/stable/settings.html#worker-connections). The default value for `worker-connections` is 1000. Maybe consider exposing this setting for tuning and explain that the maximum number of concurrent requests is `workers * worker-connections`. For applications which access a database understanding that you may be requesting a large number of simultaneously connections could be important! I mistakenly believed that max concurrent requests was `2 * num_cpu_cores + 1` when it is in fact `(2 * num_cpu_cores + 1) * 1000`

**Comment by tiangolo** [comment, marked as answer]
Thanks for the patience with my reply!

I haven't used this Docker image in a very long while, so I wouldn't be able to put the time to add new features, but thanks for the suggestion! :coffee: 

Also, there's a high chance that you don't really need it: https://github.com/tiangolo/meinheld-gunicorn-flask-docker#-warning-you-probably-dont-need-this-docker-image
